--- skinview3d-original/src/animation.ts    2020-08-25 23:11:43.000000000 -0400
+++ skinview3d/src/animation.ts    2020-08-25 23:01:10.000000000 -0400
@@ -151,8 +151,8 @@
     skin.rightArm.rotation.z = Math.cos(time + Math.PI) * 0.03 - basicArmRotationZ;

     // Head shaking with different frequency & amplitude
-    skin.head.rotation.y = Math.sin(time / 4) * 0.2;
-    skin.head.rotation.x = Math.sin(time / 5) * 0.1;
+    //skin.head.rotation.y = Math.sin(time / 4) * 0.2;
+    //skin.head.rotation.x = Math.sin(time / 5) * 0.1;

     // Always add an angle for cape around the x axis
     const basicCapeRotationX = Math.PI * 0.06;